docs/project-backlog.md

```diff
@@ -0,0 +1,122 @@
+
+* If a PR is merged which resolves the issue, the ticket can be closed as `completed`.
+* If work is blocked on review, the ticket transitions to `needs-review`.
+* If work is abandoned, or the ocntributors otherwise change their minds, the ticket can be closed as `no-further-action`.
+
+### Needs Review
+label: `needs-review`
+
+Implementation work related to the ticket has been completed. Work related to a Review and/or any changes requested as part of the review  is in progress.  Issues with a PR associated with it will be considered as in ‘In Review’.
+
+* Move the issue back to `in-progess` if significant changes are needed.
+* Close the issue with the `completed` label.
+* Close the issue with the `no-action` label.
+
+### Needs Discussion
+label: `needs-discussion`
+
+Some issues are not directly related to a particular code change. This can often arise with tickets describing unclear edge cases, and situations where applying OpeTelemetry is proving to be difficult.  If a ticket is worth considering in the issue backlog, but not scoped clearly enough to move to `wip`, then it `needs-discussion`.
+
+* When possible, move the discussion forwards by using code examples.
+* Mark the ticket as `agreed` if a resolution is found. 
+* Close the ticket as `no-action` if discussion ends without a solution.
+* Note: If the ticket is a help request with an existing answer, please link to the answer before closing the ticket.
+
+### Agreed
+label: `agreed`
+
+An agreement has been reached. Once a course of action is determined, discussion is (hopefully) over.
+
+* Clearly list any resulting action items which need to be adressed.
+* Close the ticket once all action items are completed.
+
+### Stale
+label: `stale`
+
+When a ticket has violated an SLA, it becomes stale.
+
+Action items for stale tickets:
+* record the SLA details as comment on the ticket.
+* @mention the person(s) who can resolve the SLA.
+
+### Blocked
+label: `blocked`
+
+An Issue is blocked due to an external reason or extenuating circumstances. Genereally, issues should be closed as `no-further-action`, rather than be left in a `blocked` state.
+
+### No Futher Action
+label: `no-further-action`
+
+Not every discussion or feature request results in action. Issues which cannot be adressed at the present moment, or have no futher discussion, should be closed. This helps keep the open backlog focused on active discussions.
+
+Possible reasons:
+* a prior discussion decided the issue for now.
+* a discussion is off topic.
+* a feature request is not implementable.
+
+In these cases, close the ticket as `no-further-action`. Please include a description explaining why the ticket was closed, including links to prior tickets if available. Please describe how the ticket can be reopened if needed.
+
+### Completed
+label: `completed`
+
+Any issues which are resolved via a merged pull request can be closed with the `completed`. This extra label can be helpful when searching closed issues.
```

Comment:
can you please elaborate on it? So you want to differentiate in search wether issue was `completed` or closed with explanation of user error? Typically you don't care, just read comments and associated PRs should be enough

Reply:
Mainly, this is here because, like “new,” completed is an important state. 

I agree that we can use “closed with no label” to mean “completed” if it is too difficult to add a label. 

As to why the label is an important category, when searching the past it is important to scope search to work that was incorporated into the project, versus off-topic discussions or rejected features. 

Closed labels:
* record of work. (completed)
* record of non-work requests. (no-further-action)
* record of issues closed without due process. (no label)

Reply:
I'm curious what kind of a search do you have in mind? Looking for some exception from SDK, but only when it resulted in "completed" PR? I see how it can be useful for analytics. Perhaps we can identify what kind of analytics we need to run to inform what kind of additional work to impose on maintainers